src/main/kotlin/com/meltwater/docker/compose/DockerCompose.kt

```diff
@@ -55,21 +57,44 @@ class DockerCompose private constructor(
 
         private val ENVIRONMENT_VARIABLE_SEPARATOR = Pattern.compile("=")
 
-        private fun initializeNewDockerComposeSession(classPathYmlResource: String, env: HashMap<String, String>): String {
+        private val dockerCompose: String = findDockerCompose("docker-compose")?.toString() ?: "docker compose"
```

Comment:
What will happen if you don't have docker-compose installed, and the version of docker is too old to have the compose argument/feature?

Reply:
It will basically fail because the `docker` command will fail. It is not different from what happens f you don't have `docker-compose` installed.